src/easydmp/dmpt/models.py

```diff
@@ -438,17 +438,16 @@ def _make_do_section_question(self):
         with transaction.atomic():
             do_section_question = DoSectionQuestion(section=self,
                                                     question=self.optional_text,
-                                                    position=-1,
-                                                    branching_possible=True)
+                                                    position=0)
+            do_section_question.save()
             yes = CannedAnswer(question=do_section_question, canned_text='Yes')
+            yes.save()
             no = CannedAnswer(question=do_section_question, canned_text='No')
+            no.save()
             branch_past_section = ExplicitBranch(current_question=do_section_question,
```

Comment:
ExplicitBranch only points to questions in their own section, never to questions in other sections.